iai_apartment/launch/kitchen2_display.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+
+	<arg name="model" default="$(find iai_apartment)/urdf/kitchen.urdf" />
+	<param name="apartment_description" command="$(find xacro)/xacro '$(arg model)'" />
+	<node name="apartment_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
+		<remap from="robot_description" to="apartment_description" />
+		<remap from="joint_states" to="apartment_joint_states" />
+		<param name="tf_prefix" value="iai_apartment" />
+		<param name="publish_frequency" value="60" />
+	</node>
+	<node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
+		<remap from="robot_description" to="apartment_description" />
+		<remap from="joint_states" to="apartment_joint_states" />
+	</node>
```

Comment:
Only use this in the _display.launch files. In the _upload.launch, this should be replaced with

    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <remap from="robot_description" to="apartment_description" />
        <remap from="joint_states" to="apartment_joint_states" />
        <rosparam param="source_list">
            - /kitchen/cram_joint_states
        </rosparam>
    </node>

the source list allows us to set the joint state of the kitchen via topic